packages/dynamic-import-vars/rollup.config.js

```diff
@@ -4,7 +4,7 @@ export default {
   input: 'src/index.js',
   external: [...Object.keys(pkg.dependencies), 'path'],
   output: [
-    { file: pkg.main, format: 'cjs' },
+    { file: pkg.main, format: 'cjs', exports: 'named' },
```

Comment:
is this due to new rollup features?

Reply:
This one is actually an old feature: If there is a default and named exports, Rollup switches to named exports mode but displays a warning. This was the case for this plugin, making it explicit silences the warning.

Reply:
Thanks!